annlabmed.csl

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
+  <info>
+    <title>Annals of Laboratory Medicine</title>
+    <id>http://www.zotero.org/styles/annlabmed</id>
+    <link href="http://www.zotero.org/styles/annlabmed" rel="self"/>
+    <link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
+    <author>
+      <name>Jun Hyung Lee</name>
+      <email>hermes2014@gmail.com</email>
+    </author>
+    <contributor>
+      <name></name>
+      <email></email>
+    </contributor>
+    <contributor>
+      <name></name>
+    </contributor>
```

Comment:
please add the ISSN(s). See here: https://github.com/citation-style-language/styles/wiki/Style-Requirements#9---issn-and-eissn